va/va_test.go

```diff
@@ -532,11 +532,7 @@ func TestMultiVA(t *testing.T) {
 			// Configure a primary VA with testcase remote VAs.
 			localVA, mockLog := setup(ms.Server, 0, localUA, tc.RemoteVAs)
 
-			if tc.Features != nil {
-				err := features.Set(tc.Features)
-				test.AssertNotError(t, err, "Failed to set feature flags")
-				defer features.Reset()
```

Comment:
I noticed that you removed this defer because it doesn't appear to be needed anymore.

Reply:
Ah thanks for pointing this out, this removal actually wasn't intentional. Calling Reset is now _less_ necessary, since each call to Set is a full overwrite instead of augmenting whatever has already been set, but we still shouldn't let the last one of these test cases accidentally leave features set for whichever test runs next.